### PR TITLE
Add missing production files

### PR DESCRIPTION
**Pull request by ssoloff**
These files should have been added in #2 to achieve a perfect match with production.  They don't appear to be used for anything and can probably be safely ignored.  However, I'm including them now for the following reasons:

1. _dice.class.old.php_ probably has some historical context, so it would be nice to get it into source control before we blow it away in production.  Note that this file contained sensitive data (DB connection information).  I replaced those values with the string `"REDACTED"`.  In all other respects, it is identical to the current file in production.
1. I'm writing a deployment script for the new MARTI, and I want to ensure it behaves correctly with the _exact_ content of the current MARTI folder in production.

These files will ultimately be removed in a future PR once the new MARTI is deployed.

### CODE REVIEW

File: dice/.well-known/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```
Comment (ssoloff) [inline dice/.well-known/.gitignore:2]: @RoiEXLab After further research, it appears the _.well-known_ directory is an artifact created by Nginx because we're hosting the _dice_ folder as the root of the subdomain `dice.tripleawarclub.org`.  Can you confirm?  (There's a similar folder in the parent directory, which is the root of the `www.tripleawarclub.org` subdomain.)

If so, then there's no reason to track that in this repo, and I'll remove it.

Comment (RoiEXLab) [reply dice/.well-known/.gitignore:2]: I believe the well-known folder is an artifact of the let's encrypt authentication process, and therefore generated automatically, so indeed not needed.

File: dice/dice.class.old.php [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+		else {
+		   echo("<p>Email delivery failed...</p> Dice results were not sent. <br> Please try it later again.");
+		   exit("<p>fatal error: email delivery failed!");
+		}
+
+	}
+
+
+}
+?>
```
Comment (RoiEXLab) [inline dice/dice.class.old.php:340]: The dice.class.old.php file used to be the main  dice.class.php file and the current one was the dice.class.new.php file. 
When I upgraded the server and PHP 7 was required, the old script no longer worked, but luckily there was already a new script compatible with PHP 7.
TL;DR This file is unused and broken, it can be removed in another commit

**Review by RoiEXLab — APPROVED** [review]

File: dice/.well-known/.gitignore (hunk 1 above)
Comment (ssoloff) [reply dice/.well-known/.gitignore:2]: Thanks.  Will remove in a follow-up PR.
